### DISCUSSION TITLE
Sensor Description name change

### DISCUSSION THREAD

**Discussion by conmank3** [Security-Onion-Solutions/securityonion#8750] [General] [answered]
Hello all, I recently installed a sensor on our network and fudged up the description name. I have been searching to try and find a .conf file to adjust the name, but haven't had any luck. Can anyone point me in the right direction to adjusting the description of the sensor before going through the install again? 

Much appreciated!

**Comment by njohnson-q** [comment, marked as answer]
Good day,

The files will be listed in `/opt/so/saltstack/local/pillar/minions/`. Look for the name of the node in that directory by using something like `ls -la` it will follow this format: `node-name_node-type.sls` in that file there should be a field called `node_description` (mine is sitting around line 6). Make the changes needed and then run `sudo salt '*' state.highstate`. Once completed your description should be updated.

Thank you,
Nick

**Reply by conmank3** [reply]
Thank you for the help. Did exactly as you said and it worked like a charm.
